CONTRIBUTING.md

```diff
@@ -109,6 +109,9 @@ Core developers should read the material on the [LinkML site](https://linkml.io/
     - Follow the naming conventions of the parent class
     - Descriptions of child classes may reference parent classes in a genus-differentia definition structure (e.g. "A workflow execution activity that...")
     - Inheritance should be monotonic: `slot_usage` should refine rather than override
+- ID patterning and checks
+    - ID patterns for new classes should follow conventions found [here](https://microbiomedata.github.io/nmdc-schema/identifiers/)
+    - Class-linking slots (i.e. `has_input`) should have slot_usage declared that limit range to only instances of the expecting linked class (i.e. `syntax: "{id_nmdc_prefix}:chrcon-{id_shoulder}-{id_blade}$"`)
```

Comment:
You're talking about the `structured_pattern`s here, right? The range is limited by the `range` metaslot, and not anything else. This is also intended as a short term fix, just until the LinkML developers come up with a more general solution that doesn't require us to make repetitive assertions.

Reply:
Yes, this is in reference to the `structured_pattern` use.  I understand that its a temporary fix, but it's been a big pain for the re-IDing team so I wanted to document it somewhere until linkML can verify ranges.

I will update language to be more explicit.